test/common/upstream/health_check_fuzz.h

```diff
@@ -45,6 +47,14 @@ class HttpHealthCheckFuzz : public HealthCheckFuzz, HttpHealthCheckerImplTestBas
   void triggerIntervalTimer(bool expect_client_create) override;
   void triggerTimeoutTimer(bool last_action) override;
   void raiseEvent(const Network::ConnectionEvent& event_type, bool last_action) override;
+  // No op
+  void raiseGoAway(bool no_error) override {
```

Comment:
You can just remove the `no_error` name and then the function body.

Reply:
Oh nice thanks for the tip!